pkgs/top-level/all-packages.nix

```diff
@@ -15296,9 +15296,11 @@ with pkgs;
   apacheAnt_1_9 = callPackage ../development/tools/build-managers/apache-ant/1.9.nix { };
   ant = apacheAnt;
 
-  apacheKafka = apacheKafka_2_8;
-  apacheKafka_2_7 = callPackage ../servers/apache-kafka { majorVersion = "2.7"; };
+  apacheKafka = apacheKafka_3_2;
```

Comment:
Do we need a changelog entry for the bump?

Reply:
Personally we lock this to the specific version, as would hope most shops do, since you can't just upgrade this without doing rolling version management and setting a version parameter in the properties. So it shouldn't impact anyone.

Reply:
I agree with this reasoning. We could do with a better sunset policy (I've been very inconsistent, personally) such that every release keeps at least the _newest_ release from the prior NixOS version (though it's not strictly necessary due to leniency in kafka upgrades) but it really should be pinned even within a single kafka version. Thanks!